support-frontend/assets/pages/subscriptions-redemption/components/productSummary/productSummaryStyles.js

```diff
@@ -18,8 +18,8 @@ export const imageContainer = css`
   display: inline-flex;
   align-items: flex-start;
   width: calc(100% - 30px);
-  padding: ${space[4]}px ${space[4]}px 0 ${space[4]}px;
-  background-color: ${neutral['60']};
+  padding: ${space[4]}px ${space[4]}px 0;
+  background-color: ${neutral['97']};
```

Comment:
I think you can import this as a colour token (https://theguardian.design/2a1e5182b/p/75f7ca-dark-palette, and https://theguardian.design/2a1e5182b/p/1377a6-tokens/b/293ddb) rather than a global colour. Like `${background.secondary}` and you import it as follows
```
import { background } from "@guardian/src-foundations/palette"
```

Reply:
Thanks, that is nicer!